.pre-commit-config.yaml

```
@@ -5,6 +5,13 @@ repos:
   - id: end-of-file-fixer
   - id: trailing-whitespace
   - id: check-added-large-files
+  - id: mixed-line-ending
+    args: [--fix=lf]
+    exclude_types: [batch]
+  - id: mixed-line-ending
+    args: [--fix=crlf]
+    types: [batch]
+
```

I don't think this repository will ever contain batch files, can we just force LF for every files please?

```suggestion
  - id: mixed-line-ending
    args: [--fix=lf]**
```

---

it does https://github.com/Delgan/loguru/blob/master/docs/make.bat

---

Haha, my bad.

---

Yes, I agree. I should have run `sphinx-build` with the `--no-batchfile` argument.